bindings/protocols/coap/index.html

```diff
@@ -1,181 +1,464 @@
+                        <td>Indicates that a quick block-wise transfer is supported [[I-D.ietf-core-new-block]]</td>
+                        <td>optional</td>
+                        <td><a href="#BlockWiseTransferParameters">QuickBlockWiseTransferParameters</a></td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:hopLimit</code></td>
+                        <td>Indicates that a CoAP Hop-Limit option [[RFC8768]] be included in the request</td>
+                        <td>optional</td>
+                        <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+                    </tr>
+                </tbody>
+            </table>
+        </section>
+        <section id="Method">
+            <h3>Method</h3>
+            <p>
+                ... including (but not limited to) the following methods:
+            </p>
+            <table class="def" title="Method">
+                <thead>
+                    <tr>
+                        <th>Value</th>
+                        <th>Description</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><code>coap:get</code></td>
+                        <td>The GET method [[RFC7252]]</td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:post</code></td>
+                        <td>The POST method [[RFC7252]]</td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:put</code></td>
+                        <td>The PUT method [[RFC7252]]</td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:delete</code></td>
+                        <td>The DELETE method [[RFC7252]]</td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:fetch</code></td>
+                        <td>The FETCH method [[RFC8132]]</td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:patch</code></td>
+                        <td>The PATCH method [[RFC8132]]</td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:ipatch</code></td>
+                        <td>The iPATCH method [[RFC8132]]</td>
+                    </tr>
+                </tbody>
+            </table>
+            <p>
+                See
+                <a href="https://www.iana.org/assignments/core-parameters/#method-codes">Constrained RESTful Environments (CoRE) Parameters</a>
+                for a complete list of methods.
+            </p>
+        </section>
+        <section id="ContentFormat">
+            <h3>ContentFormat</h3>
+            <p>
+                ... including (but not limited to) the following content formats:
+            </p>
+            <table class="def" title="ContentFormat">
```

Comment:
Since this table is essentially a copy of the IANA registry, should we have it at all?

Reply:
I think a reference to IANA would probably be the most elegant solution. Maybe an example for the use of the option values could be given instead?